poolqc/poolingQC_sample.py

```diff
@@ -0,0 +1,300 @@
+import dash_core_components as dcc
+import dash_html_components as html
+import dash.dependencies as dep
+import dash_table as dt
+import pandas
+import plotly.plotly as plotly
+import plotly.graph_objs as go
+import numpy as np
+
+pandas.options.mode.chained_assignment = None
+index = pandas.read_hdf('/home/ijafari/cache_files/rnaseqqc_cache.hd5')
```

Comment:
Please don't hard-code paths here, as nobody else's computer will have a `/home/ijafari` folder on it! You should be able to use relative paths as you know the rest of the dashi project expects to find cache files in a `data` folder located at the root of the dashi project.

Reply:
Or use `argparse` so the directories can be specified on the command line.

Reply:
@apmasell I don't know if that'll work when Dash is running in web server mode...